cumulusci/core/tests/untrusted_repo_parent/cumulusci.yml

```diff
@@ -0,0 +1,22 @@
+project:
+    name: Untrusted-Repo-Parent
+
+sources:
+    # trusted by the parent but not the root cumulusci
+    untrusted_child:
+        path: cumulusci/core/tests/untrusted_repo_child
+        allow_remote_code: True
+
+tasks:
+    example_task:
+        description: An example project-level task that does nothing
+        class_path: tasks.untrusted_parent.ExampleTask
+
+flows:
+    untrusted:
+        steps:
+            1:
+                task: example_task
+
```

Comment:
Is this spacing intentional?

Reply:
I'm not sure if it changed since you commented. I do sometimes use a newline between flow steps. It would be best to be consistent but I didn't notice until you mentioned it.